### PR TITLE
Implemented Deadline Command with Junit tests

### LINKED ISSUE #15: Implement "deadline command"
The deadline command sets or changes the deadline of a user's personal event

### PR DESCRIPTION
**Pull request by ninggggx99**
Fixes #15

### CODE REVIEW

File: src/main/java/seedu/duke/command/DeadlineCommand.java [hunk 1]
```diff
@@ -18,8 +27,62 @@ public DeadlineCommand(String command) {
...
+                String timeString = commandSplit[2].trim();
+                timeString = timeString.replace(":", "");
+                time = DateTimeParser.timeParser(timeString);
+            } catch (Exception e) {
+                ui.printExceptionMessage(e.toString());
+            }
+
+            return 1;
+        }
+        return 2;
```
Comment (marcursor) [inline src/main/java/seedu/duke/command/DeadlineCommand.java:85]: Maybe can use constants to prevent magic numbers

**Review by marcursor — APPROVED** [review]
Good job, well documented with Javadoc

**Review by Reinbowl — CHANGES_REQUESTED** [review]
Using \r in JUnit test only works for Windows. To make test platform independent try using StringWriter and PrintWriter.

**Comment by marcursor** [comment]
> Using \r in JUnit test only works for Windows. To make test platform independent try using StringWriter and PrintWriter.

maybe can try using System.lineSeparator() for line breaks?

**Review by Reinbowl — APPROVED** [review]

File: src/main/java/seedu/duke/command/DeadlineCommand.java (hunk 1 above)
Comment (matthewgani) [reply src/main/java/seedu/duke/command/DeadlineCommand.java:85]: Agreed! Would make things more readable/understandable

**Pull request merged**